test/12-device-config.spec.ts

```diff
@@ -171,6 +174,54 @@ describe('Device Backend Config', () => {
 		(child_process.exec as SinonStub).restore();
 	});
 
+	it('ensures required fields are written to config.txt', async () => {
```

Comment:
This file places any tests that use a backend into their own suite except for config-txt. The first 5 cases in this file are using config-txt. I think it would be nice to tidy it up and put them into their own suite like the other backend specific device config cases

Reply:
I don't love this either, but the counterargument I see is, these are more like integration tests from the device config side of the code, where the backend tests are checking the actual backend behavior. 

There probably should be a test for the config backend, then these tests could become much smaller, but probably not going to address that in this PR.